src/pages/components/Overlay.qml

```diff
@@ -158,6 +158,29 @@ PanelBackground {
         source: "image://theme/graphic-gradient-edge"
     }
 
+    Label {
+        id: privateModeLabel
+
+        PrivateModeTexture {
+            anchors.fill: parent
```

Comment:
Not filling the space above overlay. Visible when dragging overlay down.

Should be filling Rectangle @ BrowserPage.qml 123. Could be added as a children of it.

From out side of the Overlay the texture is active when on private mode and toolBar not visible...
```
active: !overlay.toolBar.visible && webView.privateMode
```